NEXT_CHANGELOG.md

```diff
@@ -166,6 +166,23 @@ By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollo
 
 ## 📚 Documentation
 
+### Add documentation for `send_headers` and `send_variable_values` in `telemetry.apollo` ([Issue #2149](https://github.com/apollographql/router/issues/2149))
```

Comment:
I don't think we need to say "Add documentation" in each Documentation changelog, but no hard feelings.

```suggestion
### `send_headers` and `send_variable_values` in `telemetry.apollo` ([Issue #2149](https://github.com/apollographql/router/issues/2149))
```